src/apis/getItem.ts

```diff
@@ -0,0 +1,10 @@
+import { API_KEY } from '@/constants/constantValues';
+
+export const getItem = (contentType: string, id: string) => {
+  const ITEM_URL = `https://api.themoviedb.org/3/${contentType}/${id}?api_key=${API_KEY}&append_to_response=videos`;
```

Comment:
Move `https://api.themoviedb.org/3` to .env file. Name it as `API_MOVIES_BASE_URL`.